README.md

```diff
@@ -72,20 +69,17 @@ pip install jina
 
 Find more install options on [Apple Silicon](https://docs.jina.ai/get-started/install/apple-silicon-m1-m2/)/[Windows](https://docs.jina.ai/get-started/install/windows/).
 
-
 ## Get Started
 
-
 ### Basic Concepts
 
-Document, Executor and Flow are three fundamental concepts in Jina.
-
-- [**Document**](https://docarray.jina.ai/) from [DocArray](https://github.com/docarray/docarray) is the basic computational unit in Jina.
-- [**Executor**](https://docs.jina.ai/concepts/executor/) is a Python class that transforms and processes Documents.
-- [**Flow**](https://docs.jina.ai/concepts/flow/) and [**Deployment**](https://docs.jina.ai/concepts/executor/serve/#serve-directly) orchestrates Executors into standalone services or pipelines.
+Jina has three fundamental concepts: Document, Executor and Flow:
```

Comment:
We need to expose Deployment here.

Reply:
OK. I just removed the `Document, Executor and Flow:` part, since we list them straight afterwards. Want to keep this DRY